examples/typescript/platform-ref-aws/crossplane.ts

```diff
@@ -0,0 +1,77 @@
+    });
+
+    config.addProvider('registry.upbound.io/crossplane/provider-aws', '>=v0.14.0-0');
+    config.addProvider('registry.upbound.io/crossplane/provider-helm', '>=v0.3.6-0');
+
+    return yaml;
+}
+
+export function postgresYaml(crossplanePackage: App): Chart {
+    const yaml = new Chart(crossplanePackage, 'postgres-api');
+    xrds.compositepostgresqlinstancesAwsPlatformrefCrossplaneIo(yaml, 'postgres-xrd');
```

Comment:
This is still not the final thing right?

Reply:
Nope just getting it working with the latest version of the reference platform. Should have some time tomorrow to get some better L2 constructs in.